zerver/lib/email_mirror.py

```diff
@@ -187,6 +187,11 @@ class ZulipEmailForwardError(Exception):
     pass
 
 def send_zulip(sender: str, stream: Stream, topic: str, content: str) -> None:
+    if len(topic) > MAX_TOPIC_NAME_LENGTH:
+        try:
+            Message.objects.get(subject=topic[:MAX_TOPIC_NAME_LENGTH])
```

Comment:
This test is wrong -- it'll check for any message in the database, not just ones to this stream.